utils/fileutils.go

```diff
@@ -21,8 +21,11 @@ const (
 	maxFileAge = 24.0
 )
 
-func IsFileExists(path string) (bool, error) {
-	fileInfo, err := os.Stat(path)
+// Check if path points at a file.
+// If path points at a symlink and `preserveSymLink == true`,
+// function will return `true` regardless of the symlink target
+func IsFileExists(path string, preserveSymLink bool) (bool, error) {
```

Comment:
If I understand correctly, it is not "preserve", but more like "follow", not? It is a read operation.
```suggestion
func IsFileExists(path string, followSymlink bool) (bool, error) {
```